<launch>
  <!-- traffic_light detection -->
  <node pkg="turtlebot3_auto_detect" type="detect_traffic_light.py" name="detect_traffic_light" output="screen">
    <remap from="/detect/image_input" to="/camera/image_compensated" />
    <remap from="/detect/image_input/compressed" to="/camera/image_compensated/compressed" />
    <remap from="/detect/image_output" to="/detect/image_traffic_light" />
    <remap from="/detect/image_output/compressed" to="/detect/image_traffic_light/compressed" />
  </node>
</launch>